integrated_channels/cornerstone/exporters/content_metadata.py

```diff
@@ -50,3 +50,17 @@ def transform_description(self, content_metadata_item):
         if 0 < len(full_description) <= self.LONG_STRING_LIMIT:  # pylint: disable=len-as-condition
             return full_description
         return content_metadata_item.get('short_description') or content_metadata_item.get('title') or ''
+
+    def transform_subjects(self, content_metadata_item):
+        """
+        Return the transformed version of the course subject list.
+        """
+        from integrated_channels.cornerstone.models import CornerstoneGlobalConfiguration
```

Comment:
why we have local import here

Reply:
Because we have imported `CornerstoneContentMetadataExporter` in models. If we import it globally its a case of circle import and will get an error.

Reply:
Then we should use this method to get `CornerstoneGlobalConfiguration` model as we are doing at other places
`apps.get_model('cornerstone', CornerstoneGlobalConfiguration)`